wsgiservice_restplus/fields.py

```diff
@@ -92,8 +92,8 @@ class Raw(object):
         than the publicly named value.
     :param str title: The field title (for documentation purpose)
     :param str description: The field description (for documentation purpose)
-    :param bool required: Is the field required ?
-    :param bool readonly: Is the field read only ? (for documentation purpose)
+    :param bool required: Is the field mandatory?
```

Comment:
If we change it to required, shouldn't we change it everywhere? eg. `self.required`, docstring above, etc.